staging/src/k8s.io/component-base/logs/json/register/register_test.go

```diff
@@ -35,13 +35,8 @@ func TestJSONFlag(t *testing.T) {
 	o.AddFlags(fs)
 	fs.SetOutput(&output)
 	fs.PrintDefaults()
-	want := `      --experimental-logging-sanitization   [Experimental] When enabled prevents logging of fields tagged as sensitive (passwords, keys, tokens).
-                                            Runtime log sanitization may introduce significant computation overhead and therefore should not be enabled in production.
-      --logging-format string               Sets the log format. Permitted formats: "json", "text".
-                                            Non-default formats don't honor these flags: --add_dir_header, --alsologtostderr, --log_backtrace_at, --log_dir, --log_file, --log_file_max_size, --logtostderr, --one_output, --skip_headers, --skip_log_headers, --stderrthreshold, --vmodule, --log-flush-frequency.
-                                            Non-default choices are currently alpha and subject to change without warning. (default "text")
-`
-	if !assert.Equal(t, want, output.String()) {
+	want := `Permitted formats: "json", "text".`
+	if !assert.Contains(t, output.String(), want) {
 		t.Errorf("Wrong list of flags. expect %q, got %q", want, output.String())
```

Comment:
Could we update the error message to imply that we are not looking for exact match?

Reply:
Oh, it has been changed, but the push failed, it has been pushed again.
Could you please check whether it can be wrongly described in this way？
```
t.Errorf("JSON logging format flag is not available. expect to contain %q, got %q", wantSubstring, output.String())
```